zk-sdk/src/encryption/grouped_elgamal.rs

```diff
@@ -6,9 +6,9 @@
 //! A regular twisted ElGamal ciphertext consists of two components:
 //! - A Pedersen commitment that encodes a message to be encrypted
 //! - A "decryption handle" that binds the Pedersen opening to a specific public key
-//! The ciphertext can be generalized to hold not a single decryption handle, but multiple handles
-//! pertaining to multiple ElGamal public keys. These ciphertexts are referred to as a "grouped"
-//! ElGamal ciphertext.
+//!   The ciphertext can be generalized to hold not a single decryption handle, but multiple handles
```

Comment:
Nit, the first line above the diff should probably have a period at the end
`to a specific public key` TO `to a specific public key.`